frontend-react/src/hooks/UseLookupTable.test.ts

```diff
@@ -0,0 +1,63 @@
+import { renderHook } from "@testing-library/react-hooks";
+
+import { lookupTableServer } from "../__mocks__/LookupTableMockServer";
+import { LookupTables, ValueSet } from "../network/api/LookupTableApi";
+
+import useLookupTable, {
+    generateUseLookupTable,
+    GetLatestData,
+    GetLatestVersion,
+    getSenderAutomationData,
+} from "./UseLookupTable";
+
+jest.mock("@okta/okta-react", () => ({
+    useOktaAuth: () => {
+        const authState = {
+            isAuthenticated: true,
+        };
+        return { authState: authState };
+    },
+}));
+
+describe("test all hooks and methods", () => {
+    beforeAll(() => lookupTableServer.listen());
+    afterEach(() => lookupTableServer.resetHandlers());
+    afterAll(() => lookupTableServer.close());
+
+    test("test GetLatestVersion", async () => {
+        let version = await GetLatestVersion(LookupTables.VALUE_SET);
+        expect(version).toEqual(2);
+    });
+
+    test("test GetLatestData", async () => {
+        let data = (await GetLatestData<ValueSet>(
+            2,
+            LookupTables.VALUE_SET
+        )) as ValueSet[];
+        expect(data[0].name).toEqual(LookupTables.VALUE_SET);
+    });
+
+    test("test getSenderAutomationData", async () => {
+        let data = await getSenderAutomationData<ValueSet>(
+            LookupTables.VALUE_SET
+        );
+        expect(data.length).toEqual(3);
```

Comment:
similar to `GetLatestVersion`, can we test the data mapping here? Maybe we can also check that `GetLatestData` is being called with the version returned from the call to `GetLatestVersion`?

Reply:
let's add these to the To Do list for #5442 @doug-s-nava